pkg/apis/resource/extension.go

```diff
@@ -561,3 +566,70 @@ func (h Handler) RouteGetGraph(req RouteGetGraphRequest) (*RouteGetGraphResponse
 		Edges:    edges,
 	}, nil
 }
+
+func (h Handler) RouteReApply(req RouteReApplyRequest) error {
```

Comment:
How do we show in the UI, should we use redeploy?

Reply:
Deploy may be better, it is weird to show re-deploy in UI.